readme/BREAKING_CHANGES_AMENDMENTS.markdown

```diff
@@ -362,4 +362,22 @@ We're trying to replace all ServiceProxyFactory usages in portal with class Snap
 ## Alternatives
 Use com.liferay.portal.kernel.module.service.Snapshot instead.
 ----
+```
+----
+
+# 169322529677c73dbd060ea11b64a9eee56415c9
+
+Missing breaking change
+
+Correct message should be:
+```
+LPD-16492 ObjectFields with Aggregation and Formula businessType should not be indexable
+    
+# breaking
+## What modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java
+Adding new validation for Aggregation and Formula ObjectField, preventing them to be indexed.
```

Comment:
Can you review this message? I think **Aggregation** and **Formula** words shouldn't be in capital letters. Regarding **ObjectField** word I'm in doubt, can you please check other examples to check if they use the entity name like "ObjectField" instead of "object field".